client/lib/directly/test/index.js

```diff
@@ -96,22 +80,20 @@ describe( 'index', () => {
 		const name = 'Richie Rich';
 		const email = 'richie@richenterprises.biz';
 
-		it( 'does nothing if Directly hasn\'t been initialized', () => {
+		it( 'initializes Directly if it hasn\'t already been initialized', () => {
```

Comment:
I'm not really a stickler for any of this, but eslint is complaining here since no variables were used in the template literal. I remember there was some debate a while back, but folks are free to open another PR to chat about modifying our js eslint rules.

https://github.com/Automattic/wp-calypso/pull/9037#discussion_r86252527